apps/daffio/src/app/core/footer/simple-footer/simple-footer.component.ts

```diff
@@ -13,13 +13,16 @@ import {
   styleUrls: ['./simple-footer.component.scss'],
 })
 export class DaffioSimpleFooterComponent {
-  faGithub = faGithub;
-  faGitter = faGitter;
 
   @HostBinding('class.daffio-simple-footer') class = true;
 
   links: any[] = [
     { path: '/why-pwa', title: 'Why PWA' },
     { path: '/support', title: 'Support' },
   ];
+
+  socialLinks: any[] = [
+    { link: 'https://github.com/graycoreio/daffodil', title: 'Github',  icon: faGithub },
+    { link: 'https://discord.gg/BdaJVZ53sR', title: 'Discord', icon: faDiscord },
+  ];
```

Comment:
Maybe this should be exported so that the sub footer can use it as well?

Reply:
subfooter is a different design, wouldn't make sense to reuse this there.

Reply:
@griest024 I like the idea, moved into branding. @xelaint I think he's referring to the link literal string.